requirements.txt

```diff
@@ -26,3 +26,9 @@ setuptools==70.0.0
 tabulate
 torchvision==0.18.0
 unstructured-client==0.23.3
+fastapi==0.109.0
+uvicorn==0.24.0-post.1
+gunicorn==21.2.0
+# We want to move this to a common module soon to prevent this
+# this is the crawl step for stop gap for now to prevent reimplmenting the same classes.
+nv_ingest_client
```

Comment:
Once again do not like this. However, making the common module I had earlier proved to be even more painful for all the changes needed in the Dockerfile, setups, and also the extra deployment that was needed to pypi